crazyswarm2/src/teleop.cpp

```diff
@@ -36,17 +42,34 @@ class TeleopNode : public rclcpp::Node
         subscription_ = this->create_subscription<sensor_msgs::msg::Joy>(
             "joy", 1, std::bind(&TeleopNode::joyChanged, this, _1));
 
-        client_emergency_ = this->create_client<Empty>("emergency");
-        client_emergency_->wait_for_service();
+        publisher_ = this->create_publisher<geometry_msgs::msg::Twist>("cmd_vel", 10);
+        timer_ = this->create_wall_timer(2s, std::bind(&TeleopNode::execute, this));
+    
+
+        // client_emergency_ = this->create_client<Empty>("emergency");
+        // client_emergency_->wait_for_service();
 
-        client_takeoff_ = this->create_client<Takeoff>("takeoff");
-        client_takeoff_->wait_for_service();
+        // client_takeoff_ = this->create_client<Takeoff>("takeoff");
+        // client_takeoff_->wait_for_service();
 
-        client_land_ = this->create_client<Land>("land");
-        client_land_->wait_for_service();
+        // client_land_ = this->create_client<Land>("land");
+        // client_land_->wait_for_service();
+        this->declare_parameter<double>("frequency", 10.0);
+        frequency_ = this->get_parameter("frequency").as_double();
     }
 
 private:
+    
+    void execute() 
+    {
+        rclcpp::Rate rate(frequency_);
+        while (rclcpp::ok()){
+            publisher_->publish(twist);
+            rate.sleep();
+        }
```

Comment:
This should have stayed unchanged, i.e., this function just publishes your twist message. The frequency_ should be used to create the wall-timer (the first argument is the period, i.e., 1/frequency_, see https://docs.ros2.org/beta3/api/rclcpp/classrclcpp_1_1node_1_1Node.html#a1a727c1777f045074e13b2cbc6e9b0e3).